sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_util/configurations.py

```diff
@@ -116,6 +102,36 @@ def _default_sampling_ratio(configurations):
     configurations[SAMPLING_RATIO_ARG] = default
 
 
-# TODO: Placeholder for future configuration
-def _default_disable_azure_core_tracing(configurations):
-    configurations[DISABLE_AZURE_CORE_TRACING_ARG] = False
+def _default_instrumentation_options(configurations):
+    disabled_instrumentation = environ.get(
+        OTEL_PYTHON_DISABLED_INSTRUMENTATIONS, ""
+    )
+    disabled_instrumentation = disabled_instrumentation.split(",")
+    # to handle users entering "requests , flask" or "requests, flask" with spaces
+    disabled_instrumentation = [
+        x.strip() for x in disabled_instrumentation
+    ]
+
+    merged_instrumentation_options = {}
+    instrumentation_options = configurations.get(INSTRUMENTATION_OPTIONS_ARG, {})
+    for lib_name in _FULLY_SUPPORTED_INSTRUMENTED_LIBRARIES:
+        disabled_by_env_var = lib_name in disabled_instrumentation
+        options = {"enabled": not disabled_by_env_var}
+        options.update(instrumentation_options.get(lib_name, {}))
```

Comment:
Is this `update` used because we might have future instrumetation options thats not just enabled?

Reply:
Yeah